### DISCUSSION TITLE
Extend known archives types

### DISCUSSION THREAD

**Discussion by og900aero** [sxyazi/yazi#622] [4. Ideas]
Yazi doesnt knows zstd format type.

I try the following code in yazi.toml:
`{ mime = "application/zstd",            use = [ "extract", "reveal" ] },`

but hanging in task.

**Comment by sxyazi** [comment]
`application/zstd` is now built-in supported:

https://github.com/sxyazi/yazi/blob/917fee939f23b00a6ea6180f74f7f0e76afc56f9/yazi-config/preset/yazi-default.toml#L71
